packages/js/src/event-emitter/types.ts

```diff
@@ -1,104 +1,55 @@
 import type {
-  ArchivedArgs,
-  CompleteArgs,
-  FetchCountArgs,
-  FetchCountResponse,
-  FetchFeedArgs,
-  FetchFeedResponse,
+  CountArgs,
+  ListNotificationsArgs,
   Notification,
+  ListNotificationsResponse,
+  CountResponse,
   ReadArgs,
+  ArchivedArgs,
+  CompleteArgs,
   RevertArgs,
   UnarchivedArgs,
   UnreadArgs,
-} from '../feeds';
+} from '../notifications';
 import { Preference } from '../preferences/preference';
 import { UpdatePreferencesArgs } from '../preferences/types';
 import type { InitializeSessionArgs } from '../session';
 import { InboxNotification, Session, WebSocketEvent } from '../types';
 
-type NovuPendingEvent<A, O = undefined> = {
+type NovuPendingEvent<A, D = undefined> = {
   args: A;
-  optimistic?: O;
-};
-type NovuSuccessEvent<A, R> = Pick<NovuPendingEvent<A>, 'args'> & {
-  result: R;
+  data?: D;
 };
-type NovuErrorEvent<A, F = undefined> = Pick<NovuPendingEvent<A>, 'args'> & {
-  error: unknown;
-  fallback?: F;
+type NovuResolvedEvent<A, D> = NovuPendingEvent<A, D> & {
+  error?: unknown;
 };
-// three possible status of the event: pending, success, error
-type EventName<T extends string> = `${T}.pending` | `${T}.success` | `${T}.error`;
+// three possible status of the event: pending, resolved
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** ‏
```suggestion
// two possible status of the event: pending, resolved
```

Reply:
I've already updated that but in a different PR as I couldn't commit the changes to this one as I had many other changes in there.